synthesizer/process/src/cost.rs

```diff
@@ -86,6 +89,12 @@ fn execution_storage_cost<N: Network>(size_in_bytes: u64) -> u64 {
     }
 }
 
+/// Returns the fixed cost for an execution.
+/// NOTE: this constant reflects the compute cost of an execution, but is not required to be paid by the user.
+pub fn execution_fixed_cost<N: Network>() -> u64 {
```

Comment:
are there plans to extend this logic (in which case an alternative name might be more fitting)? I'm wondering about the rationale for the existence of a (non-const) function wrapper for a `const`

Reply:
I swear this was in my git staging area already: 3443c3ef7

> are there plans to extend this logic (in which case an alternative name might be more fitting)?

Unclear at the moment. Only other name I can imagine is `execution_compute_cost`?